Sort directories returned by os.walk

madsushi opened the pull request:
When Python returns directories from os.walk (which uses os.listdir),
they are in an arbitrary order on some OSes (Ubuntu, etc) per the Python
documentation. Adding a .sort() in the process resolves this issue,
ensuring that directories are sorted properly and processed in the
correct order.

sjl commented:
I've moved to Hugo for my blog, so this repo is basically abandonware at this point, especially since it was a fork of the already-abandoned old version of Hyde.

madsushi commented:
No worries, I moved over to Hugo too. :)

Thanks again for your great blog!

> On Nov 24, 2016, at 04:18, Steve Losh <notifications@github.com> wrote:
> 
> I've moved to Hugo for my blog, so this repo is basically abandonware at this point, especially since it was a fork of the already-abandoned old version of Hyde.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
